silverback/_cli.py

```diff
@@ -36,10 +43,18 @@ def _account_callback(ctx, param, val):
 
 
 def _network_callback(ctx, param, val):
-    if val:
-        os.environ["SILVERBACK_NETWORK_CHOICE"] = val
+    # NOTE: Make sure both of these have the same setting
+    if env_network_choice := os.environ.get("SILVERBACK_NETWORK_CHOICE"):
+        if val.network_choice != env_network_choice:
```

Comment:
Note: @mikeshultz noticed if you use the shortened form in the environment variable, that the selection would be fully resolved and thus be mismatching

easy fix is to just update the environment variable to the fully resolved form